modules/local/hmmrank.nf

```diff
@@ -0,0 +1,78 @@
+process HMMRANK {
+    label 'process_low'
+
+    conda (params.enable_conda ? "conda-forge::r-tidyverse=1.3.1 conda-forge::r-data.table=1.14.0 conda-forge::r-dtplyr=1.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-508c9bc5e929a77a9708902b1deca248c0c84689:0bb5bee2557136d28549f41d3faa08485e967aa1-0' :
+        'quay.io/biocontainers/mulled-v2-508c9bc5e929a77a9708902b1deca248c0c84689:0bb5bee2557136d28549f41d3faa08485e967aa1-0' } "
+
+    input:
+
+    path hmmrout
+
+    output:
+
+    path "hmmrank.out", emit: x
+    //path "version.yml", emit: versions
+
+    script:
+
+    """
+    #!/usr/bin/env Rscript
+    library(optparse)
+    library(readr)
+    library(dplyr)
+    library(dtplyr)
+    library(tidyr)
+    library(data.table)
+    library(stringr)
```

Comment:
Check which are actually needed by the code. I know that `optparse` is not, as it's for grabbing command line options. And I wonder if we really need to use `data.table` and `dtplyr`. They can make the code faster, but also requires a mulled container making upgrades more complicated. It would be great if we can use the `tidyverse` container as is. This means, however, that the code needs some changes.

Reply:
Ok, I'll try to write the code with tidyverse.